spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -102,10 +102,47 @@
     end
   end
 
+  scenario "Viewing auction list and detail view as a logged out user" do
+    create_current_auction
+    
+    # seeing auction list
+    visit "/"
+    expect(page).to have_content(@auction.title)
+    expect(page).to have_content("Current bid:")
+
+    # going to the auction detail page
+    click_on(@auction.title)
+    page.find("a[href='#{@auction.issue_url}']")
+
+    # going via another link
+    visit "/"
+    click_on("View details »")
+    page.find("a[href='#{@auction.issue_url}']")
+
+    # logging in via bid click
+    click_on("BID")
+    @bidder = create_authed_bidder
+    expect(page).to have_content("Authorize with GitHub")
+    click_on("Authorize with GitHub")
+    click_on('Submit')
+
+    # bad ui brings us back to the home page :(
+    click_on("Bid »")
+    expect(page).to have_content("Current bid:")
+
+    # fill in the form
+    fill_in("bid_amount", with: '800')
+    click_on("Submit")
+
+    # returns us back to the bid page
+    expect(page).to have_content("Current bid:")
+    expect(page).to have_content("$800.00")
+  end
+
   context "placing bids" do
     scenario "Viewing auction list and detail view as a logged out user" do
       create_current_auction
-
+      @bidder = create_authed_bidder
```

Comment:
Yes, cleaning up these feature_helpers to use let and FactoryGirl more directly is in the future I think